examples/uuid/src/main.rs

```diff
@@ -0,0 +1,47 @@
+#![feature(plugin)]
+#![plugin(rocket_codegen)]
+
+#[macro_use]
+extern crate lazy_static;
+extern crate uuid;
+extern crate rocket;
+extern crate rocket_contrib;
+
+use std::collections::HashMap;
+use uuid::Uuid;
```

Comment:
Common use case is going to be inter-op with the uuid crate. Ex, it's what the Postgres and Diesel crates use for uuid support.

So, it makes sense to actually show how one does this. (Ex, the HashMap is of the uuid::Uuid type and not UUID)